src/app/app.component.ts

```diff
@@ -21,12 +21,6 @@ export class AppComponent implements OnInit {
   ) { }
 
   ngOnInit(): void {
-    // We must be in an iframe OR opened with window.open
```

Comment:
So, with this change, a user can visit identity.bitclout.com but when they try to sign up or log in their browser nothing will happen.

Could I request that you either add support for calling `window.open(bitclout.com)` when the user completes a log in or sign up flow OR disable the redirect if some param like `?webview=true` is present?

Reply:
Once this is resolved I am happy to merge away